src/theory/uf/equality_engine.h

```diff
@@ -80,31 +80,41 @@ class EqualityEngine : public context::ContextNotifyObj {
    */
   EqualityEngine* d_masterEqualityEngine;
 
-public:
-
+ public:
   /**
    * Initialize the equality engine, given the notification class.
+   *
+   * @param constantTriggers Whether we treat constants as trigger terms
+   * @param anyTermTriggers Whether we use any terms as triggers
```

Comment:
Why not have an `enum TriggerMode` with something like `NONE`, `EXPLICIT_ONLY`, `CONSTANTS_AND_EXPLICIT`?

Reply:
I don't think this is worthwhile yet, and will cause a lot of conflicts in upstream branches unfortunately.  Perhaps after centralEe and proof-new are merged we could reconsider this?

Reply:
Yeah, I am fine with deferring this to later.